manifests/weave/boarnet/bns/genesis.json

```diff
@@ -0,0 +1,440 @@
+        "//name": "Harry",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "AD102CF85A366D726BBBCA54DBD3818B6278A66A",
+        "//name": "Ingrid",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "0EDAA1225975A3A64BC7216F9429A3EA34A757D9",
+        "//name": "Jonny",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+        "//name": "Cloe",
+        "coins": [
+          "123456789 IOV"
+        ]
+      },
+      {
+        "address": "cond:gov/rule/0000000000000002",
+        "//name": "reward fund",
+        "coins": [
+          {
+            "whole": 1,
+            "ticker": "IOV"
+          }
+        ]
+      },
+      {
+        "address": "0E95C039EF14EE329D0E09D84F909CF9EB5EF472",
+        "//name": "faucet",
+        "coins": [
+          {
+            "whole": 999999999,
+            "ticker": "IOV"
+          }
+        ]
+      },
+      {
+        "address": "B1DE2F431945B8D7CE20DDD5F43FDA9609E9D7B8",
+        "//name": "loadtester",
+        "coins": [
+          {
+            "whole": 999999999,
+            "ticker": "IOV"
+          }
+        ]
+      }
+    ],
+    "currencies": [
+      {
+        "ticker": "IOV",
+        "name": "Main token of this chain"
+      }
+    ],
+    "multisig": [
+      {
+        "activation_threshold": 1,
+        "admin_threshold": 1,
+        "participants": [
+          {
+            "signature": "BA1DBBA7C4A04F39A98FA5B4DBA8BC6981D7D40E",
+            "//name": "Cloe",
+            "weight": 1
+          }
+        ]
+      }
+    ],
+    "update_validators": {
+      "addresses": [
+        "cond:gov/rule/0000000000000004"
+      ]
+    },
+    "distribution": [
+      {
+        "//name": "unused - owned by economic committee",
+        "admin": "cond:gov/rule/0000000000000002",
+        "destinations": [
+          {
+            "address": "0E95C039EF14EE329D0E09D84F909CF9EB5EF472",
+            "weight": 1
+          }
+        ]
+      }
+    ],
+    "escrow": [
+      {
+        "//name": "guarantee fond",
+        "amount": [
+          {
+            "ticker": "IOV",
+            "whole": 1000000
+          }
+        ],
+        "arbiter": "cond:gov/rule/0000000000000002",
+        "destination": "cond:gov/rule/0000000000000002",
+        "//destination": "reward fond - cash account",
+        "source": "0000000000000000000000000000000000000000",
+        "timeout": "2999-12-31T00:00:00Z"
+      }
+    ],
+    "conf": {
+      "cash": {
+        "collector_address": "cond:gov/rule/0000000000000002",
+        "//collector_address": "reward fond - cash account",
+        "minimal_fee": "0.05 IOV"
+      },
+      "migration": {
+        "admin": "cond:gov/rule/0000000000000003",
+        "//admin": "owned by tech committee"
+      }
```

Comment:
The following diff should be available to configure usernames:
https://github.com/iov-one/iov-core/commit/12dc386e14f679c672358aadd333db6cc2cdf845
However, I am not sure what should go into `owner`